src/index.css

```diff
@@ -2,29 +2,276 @@
   box-sizing: border-box;
   margin: 0;
   padding: 0;
+  text-decoration: none;
+}
+
+:root {
+  --Montserrat: 'Montserrat', sans-serif;
+  --RobotoSlab: 'Roboto Slab', sans-serif;
+  --OffWhite: #e8e8e8;
+  --Grey: #c4c4c4;
+  --White: #fff;
+}
+
+div#root {
+  margin-top: 7rem;
+  padding-bottom: 3rem;
 }
```

Comment:
I think you are a bit mistaken. The main Book page is this snapshot below and the background color is `#fafafa`. It starts with the `Navbar` and the `login icon` for users. 
![image](https://user-images.githubusercontent.com/77106962/140576065-d196aea1-a031-4ae2-a543-ca1ddfdd146e.png)

Reply:
@Gambit142 It seems it was not 100% clear for you, let me reiterate: you are right your app starts with a navbar on the top while there should have been 9.3rem margin on the top. I intentionally hovered over Zeplin's navbar, [on my previously shared snapshot](https://user-images.githubusercontent.com/60043698/140563029-0305447c-e84c-428e-aadc-91405ed2ffbb.png), so that those margin units from top and sides can be obviously visible.
I hope this is clear:wink::pray:

Anyway your project was approved, congratulations:partying_face::clap::clap::clap:
Happy coding:mechanical_arm: